selftests/unit/test_utils_linux_modules.py

```diff
@@ -28,38 +35,178 @@ class TestLsmod(unittest.TestCase):
 ip6table_filter        12815  1
 """
 
+    PROC_MODULES_OUT = b"""snd_usb_audio 225280 0 - Live 0x0000000000000000
```

Comment:
What if we keep this list only with the modules used on tests? I think is only 4 modules being checked, right? Or is there any other reason to keep this huge list?

Reply:
I just thought that we should have a realistic example (got this from my own system).  I felt I'd cheating if I tweaked it.

Reply:
I think it would be _cheating as much as mock is_, hehe. But it's ok keep the all output. Approving it.